ml-agents/mlagents/trainers/components/reward_signals/reward_signal.py

```diff
@@ -32,6 +32,9 @@ def __init__(self, policy: TFPolicy, strength: float, gamma: float):
         short_name = class_name.replace("RewardSignal", "")
         self.stat_name = f"Policy/{short_name} Reward"
         self.value_name = f"Policy/{short_name} Value Estimate"
+        # Don't terminate discounted reward computation at Done. Useful for eliminating positive bias in rewards with
+        # no natural end, e.g. GAIL or Curiosity
+        self.ignore_terminal_states = False
```

Comment:
nit (feel free to ignore): could you name this in terms of what it does instead of what it doesn't do? I find double-negatives like this a little harder to understand when trying to reason about the code.

Reply:
It was confusing for me too while writing it - I reversed it

Reply:
Cool, not sure if it's on any official naming scheme guides, but I generally try avoid using variables with negative names for this reason.